deploy/docker/Dockerfile

```diff
@@ -17,14 +17,13 @@ RUN apk add --no-cache --update --virtual .build-deps sudo build-base ruby-dev \
        && gem install snappy
 
 # FluentD plugins from RubyGems
-RUN gem install fluent-plugin-s3 -v 1.1.4 \
```

Comment:
is this plugin not used anywhere so we are removing it?

Reply:
It's not being used currently, but after talking with Frank we want to keep it around in case a customer wants to fork their data to S3 (and in the future GCP, Azure, etc.).